courses/free-courses-ta.md

```diff
@@ -57,6 +59,13 @@ CSE Tamila by Eezytutorials
 * [Express JS Tutorial In Tamil](https://youtube.com/playlist?list=PLtMr2pEysMV6ArKDOGVmjQxjW-RcdxFHE&si=CR4PI0sjOcAUnXut) - VJ Techno Wizard
 
 
+### Flutter
+
+* [Flutter in Tamil](https://www.youtube.com/playlist?list=PLBngtsPyn30HYEkmqIqZwvLl0c8zWqCtx) - LEARN ALL IN TAMIL
+* [Flutter Tamil Tutorial](https://www.youtube.com/playlist?list=PL_hkki6Usyn4krz6wexRj1baFE-TWKVMb) - Techashonline
```

Comment:
Don't invent titles! Use the title given by the resource.

Reply:
Can you please explain me which titles are you referring to?